msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md

```diff
@@ -384,6 +386,41 @@ The following table lists the throttling limits for tag mentions in a bot:
 * Tag mentions aren't supported in connectors.
 * Tag mentions don't support the invoke flow in a bot.
 
+### Support for Microsoft Entra Object ID and UPN in user mention
```

Comment:
As suggested in the group chat, how about we move this section under User mention?
#### User mention
##### Support for Microsoft Entra Object ID and UPN

Reply:
@limamicro As suggested, I've moved the section to flow as:
- User mention
- Support for Microsoft Entra Object ID and UPN
- Tag mention

Please review and let me know if flow is okay now.